packages/web3-shared/solana/utils/token.ts

```diff
@@ -1,17 +1,15 @@
 import { createFungibleToken } from '@masknet/web3-shared-base'
-import { getTokenConstants } from '../constants'
-import { chainResolver } from './resolver'
+import { getTokenConstant } from '../constants'
 import { ChainId, SchemaType } from '../types'
 
 export function createNativeToken(chainId: ChainId) {
-    const nativeCurrency = chainResolver.nativeCurrency(chainId)
     return createFungibleToken<ChainId, SchemaType.Native>(
         chainId,
         SchemaType.Native,
-        getTokenConstants(chainId).SOL_ADDRESS!,
-        nativeCurrency?.name ?? 'Sol',
-        nativeCurrency?.symbol ?? 'SOL',
-        nativeCurrency?.decimals ?? 9,
-        nativeCurrency?.logoURL,
+        getTokenConstant(chainId, 'SOL_ADDRESS')!,
```

Comment:
What about rename 'XXXTOKEN_ADDRESS' to 'XXXTOKEN' ?

Reply:
We might be confused if we do so.